packages/core/saved-objects/core-saved-objects-server-mocks/README.md

```diff
@@ -0,0 +1,5 @@
+# @kbn/core-saved-objects-server-mocks
+
+This package contains the mocks for core's server-side savedObjects service.
```

Comment:
I'm starting to think we need a core-dev-only `README` in the root `packages/core/saved-objects` folder as an index to where all the sub-domain services and mocks are within the package architecture. Other teams may end up in similar situations and it would be a huge enhancement to have it automatically generated! Perhaps our api-docs might help or serve as a base to work from. @afharo WDYT?